test/test_ops.py

```diff
@@ -1136,6 +1136,9 @@ def test_matvecmat(self):
   def test_matvec(self):
     helper_test_op([(1,128), (128,128)], lambda x,y: (x@y).relu(), atol=1e-4)
 
+  def test_big_matvec(self):
+    helper_test_op([(1,4096), (4096,1024)], lambda x,y: (x@y).relu(), atol=1e-4)
```

Comment:
This doesn't belong here. test_ops is about correctness, not speed

Reply:
It is extraneous as the existing test hits the code path.  Removed now.